wordle-web/src/components/ActiveUser.vue

```diff
@@ -1,49 +1,76 @@
 <template>
   <v-btn @click="showDialog = true">
-    {{ playerService.player.name }}
+    <span v-if="!signInService.isSignedIn">Not signed in</span>
+    <span v-else>{{ signInService.token.userName }}</span>
   </v-btn>
-  <v-dialog :model-value="showDialog" @update:model-value="close" persistent>
-    <v-card>
-      <v-card-text>
-        Set or change your username:
-        <v-text-field ref="$input" v-model="newName" @keyup.enter="confirm" />
+  <v-dialog :model-value="showDialog" @update:model-value="close">
+    <v-card
+      :class="display.xs ? 'w-100' : display.sm ? 'w-75' : display.md ? 'w-50' : 'w-25'"
+      class="justify-center mx-auto"
+    >
```

Comment:
Is there a better way of making the size responsive - or is this already the best way?

Reply:
This is an acceptable way to do it, however, vuetify is pretty good at authorizing things if you use percentages or max-widths. Were you running into resizing issues with your login dialog?